files.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 var pythonFiles map[string][]byte
+var rubyFiles map[string][]byte
```

Comment:
Can be merged into one map. This was away from my eyes cause in IDE I just folded whole `pythonFiles` block to not to scroll it all and worked with `rubyFiles`.